PSScripts/Start-SSISIntegrationRuntime.ps1

```diff
@@ -0,0 +1,62 @@
+<#
+.SYNOPSIS
+Starts an SSIS Integration Runtime, waiting for it to be created if necessary
+
+.DESCRIPTION
+When an instance of the SSIS runtime is created, it is created in a stopped state.
+This script waits for the SSIS runtime to be created,  then starts the integration runtime
+
+.PARAMETER ResourceGroupName
+The name of the Resource Group containing the Azure Data Factory v2 instance
+
+.PARAMETER DataFactoryName
+The name of the v2 Azure DataFactory instance containing the SSIS integration runtime
+
+.PARAMETER RuntimeName
+The name of the SSIS runtime to start
+
+.EXAMPLE
+Start-SSISIntegrationRuntime.ps1 -ResourceGroupName dfc-foo-bar-rg -DataFactoryName dfc-foo-msql-df -RuntimeName ssisRuntimeName
+#>
+
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $ResourceGroupName,
+    [Parameter(Mandatory=$true)]
+    [string] $DataFactoryName,
+    [Parameter(Mandatory=$true)]
+    [string] $RuntimeName
+)
+
+$breakStates = @("Initial", "Stopped", "Started")
+
+Write-Host "Checking to see if integration runtime '$RuntimeName' has been created yet."
+Write-Host "Will retry every 15 seconds for 10 minutes before failing..."
+
+$attempts = 1
+$runtimeInstance = $null
+
+while($true) {    
```

Comment:
Should this be a "do while" rather than a "while" with the condition being $runtimeInstance.State -inotin $breakStates?